eclair-core/src/main/scala/fr/acinq/eclair/balance/CheckBalance.scala

```diff
@@ -289,7 +289,7 @@ object CheckBalance {
     }
   } yield CorrectedOnChainBalance(detailed.confirmed, detailed.unconfirmed)
 
-  case class GlobalBalance (onChain: CorrectedOnChainBalance, offChain: OffChainBalance) {
+  case class GlobalBalance (onChain: CorrectedOnChainBalance, offChain: OffChainBalance, channelsCount: Int) {
```

Comment:
I see how useful the `channelsCount` may be, but the inconsistency is making me OCD a little bit 😵

Reply:
Yeah, I tried wrapping this in a parent type, but it wasn't very pretty either...but if you prefer I can do that, at least it keeps `GlobalBalance` with only balance-related things.